### PR TITLE
Fixed some issues and removed display errors

### PR DESCRIPTION
**Pull request by Brokkolii**
accidentally formatted the whole files

### CODE REVIEW

**Comment by rufreakde** [comment]
Hi @Brokkolii ,
I will review the PR as soon as I have time for it. Will be today or tomorrow evening.

File: content.js [hunk 1]
```diff
@@ -1,93 +1,88 @@
...
-border-style: dotted;
-border-color: #f23838;
-}
+const cssRuleElement = `span[class*="price"],
+span[class="tmm-olp-links"] *
+{
+    color: transparent !important;
+    user-select: none;
+  }
 `;
```
Comment (rufreakde) [inline content.js]:
```suggestion
const blurr = useBlurr ? "text-shadow: rgba(0, 0, 0, 0.8) -10px 0px 30px !important;" : "";
const cssRuleElement = `span[class*="price"],
span[class="tmm-olp-links"] *
{
    color: transparent !important;
    ${blurr}
    user-select: none;
  }
`;
```

I like this suggestion but lets keep the blur effect as an optional and add it into the readme.

**Review by rufreakde — CHANGES_REQUESTED** [review]
prettier formatting (thats okay I switched to prettier as well)
Change of the css rule LGTM except I would like to keep the blurr effect as optional.

See the improvement I suggest.

We can merge afterwards maybe we can update the readme as well then. :)

**Comment by Brokkolii** [comment]
Yeah, you are right with the blur! i was just playing around :)

**Review by rufreakde — APPROVED** [review]
lgtm

**Comment by rufreakde** [comment]
merging

**Pull request merged**